packages/contract-tests/test/lib/utils/ens.js

```diff
@@ -26,18 +26,22 @@ const methodABI = {
   type: "function"
 };
 const args = ["my.ens.name", 555, { from: "the.other.name" }];
+const ensSettings = {
```

Comment:
So having looked through the rest of the code I'm a bit confused.  This `ensSettings` has a `networkId` field, but everywhere else, it seems like `networkId` is separate from `ensSettings` or `ens`, and isn't pulled from it, and any potential `ensSettings.networkId` or `ens.networkId` is ignored.  Meanwhile things do look inside `ensSettings` or `ens` for `registryAddress`, which isn't here.  I'm confused as to what's going on here and why this doesn't seem to match the rest of the code.  Am I misunderstanding?

Reply:
Good catch, I think this was accidentally left in from a previous iteration of this. `ensSettings` is really just a reference to `config.options.ens` which should only contain `registryAddress` and `enabled`. I'll go ahead and update these.